docs/sources/features/datasources/loki.md

```diff
@@ -111,11 +124,16 @@ log message you're interested in.
 
 ## Templating
 
-Template variables are not yet supported by Loki.
+Instead of hard-coding things like server, application and sensor name in you metric queries you can use variables in their place. Variables are shown as dropdown select boxes at the top of the dashboard. These dropdowns makes it easy to change the data being displayed in your dashboard.
+
+Checkout the [Templating](/reference/templating) documentation for an introduction to the templating feature and the different types of template variables.
 
 ## Annotations
 
-Annotations are not yet supported by Loki.
+You can use any non metric Loki query as a source of annotations. Log content will be used annotation text so
```

Comment:
Not sure if it makes sense to mention `non metric Loki query` here but seems like it's going to be important distinction.

Reply:
non-metric should be hyphenated

Reply:
Correction:
> ... source for annotations ...

Reply:
- Missing 'as'?
- Missing comma

> Log content will be used as annotation text, so there is no need for additional mapping.